src/App.css

```diff
@@ -1,38 +1,226 @@
-.App {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,700;0,900;1,100;1,300&family=Quicksand:wght@300;400;500;600;700&family=Saira:ital,wght@0,100;0,200;0,400;0,500;0,600;1,100;1,300&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=for%20your%20website).-,Typography%3A%20font%20face%2C%20size%20and%20weight.,-Layout%3A%20composition%20and), Please use the correct font families (`gill sans` for title and body and `lato` as control font) to match the given design.

![image](https://user-images.githubusercontent.com/68796232/225100405-3eb74b77-6981-4f7e-bdf3-55ee20940adc.png)

### Other than that you have done a great job 👍.

Reply:
working on it sir, thank you very much for the review